dex/testing/firo/harness.sh

```diff
@@ -335,9 +337,9 @@ if [ -z "$NOMINER" ] ; then
   tmux send-keys -t $SESSION:5 "watch -n 15 ./mine-alpha 1" C-m
 fi
 
-# Live stop/start
-#tmux send-keys -t $SESSION:5 C-c C-m
-#tmux send-keys -t $SESSION:5 "watch -n 15 ./mine-alpha 1" C-m
+# Live stop/start - experimental
+# tmux send-keys -t $SESSION:5 C-c C-m
+# tmux send-keys -t $SESSION:5 "watch -n 15 ./mine-alpha 1" C-m
```

Comment:
What's the purpose of these changes?

Reply:
This one is easier turn off and on mining .. it has been there a while but I can remove